conda/core/path_actions.py

```diff
@@ -236,6 +237,31 @@ def reverse(self):
         rm_rf(self.target_full_path)
 
 
+class CreateApplicationEntryPointAction(CreatePathAction):
+
+    def __init__(self, transaction_context, package_info, target_prefix, target_short_path,
+                 private_env_prefix, app_name, root_prefix):
+        super(CreateApplicationEntryPointAction, self).__init__(transaction_context, package_info,
+                                                           None, None,
+                                                           target_prefix, target_short_path)
+        self.private_env_prefix = private_env_prefix
+        self.app_name = app_name
+        self.root_preifx = root_prefix
+
+    def verify(self):
+        pass
+
+    def execute(self):
+        log.trace("creating entry point %s", self.target_full_path)
```

Comment:
make this `"creating application entry point %s"`.  And then make sure the python version is `"creating python entry point %s"`